src/app.css

```diff
@@ -64,6 +63,11 @@ li {
 	list-style-type: none;
 }
 
+hr {
+	border-color: black;
```

Comment:
Where you're making changes that are wider than what the title of the commit says (in this case border colour) it'd be useful to have more of a commit message to explain why

Reply:
This hr was an small extra addition I needed, suppose I should have added it as an additional commit